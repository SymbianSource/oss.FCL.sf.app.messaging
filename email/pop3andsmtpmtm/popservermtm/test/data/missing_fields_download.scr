+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 9 33731
+OK 9 messages
1 4954
2 1776
3 6941
4 2831
5 5090
6 5616
7 3211
8 1566
9 1746
.
+OK 9 messages
1 dismbody.956722223:20:27393:1
2 dismbody.956655594:20:15333:2
3 dismbody.956621818:20:21131:2
4 dismbody.956594662:20:02578:8
5 dismbody.956660927:20:10907:20
6 dismbody.955724815:10:07960:19
7 dismbody.955053836:11:23332:2
8 dismbody.955643945:10:16672:35
9 dismbody.955354698:10:23626:0
.
-ERR Unknown Command "CAPA"
** Read cancelled
+OK 
Received: from punt-1.mail.demon.net by mailstore
          for gallery@dismbody.demon.co.uk id 955354698:10:23626:0;
          Mon, 10 Apr 2000 08:18:18 GMT
Received: from www8.masterweb.it ([192.106.185.8]) by punt-1.mail.demon.net
           id aa1128685; 10 Apr 2000 8:18 GMT
Received: from IHWU59382  [172.128.224.172] by www8.masterweb.it
  (SMTPD32-5.05) id AA7679A0478; Mon, 10 Apr 2000 10:10:14 +0100
DATE: 10 Apr 00 4:22:27 AM
FROM: 
Message-ID: <SYNO0yiL8YgZ15FI0JPv@CBXokP7jeuV1>
SUBJECT: What every business person should know

Market your Web site, product or service!
.
** Read cancelled
+OK