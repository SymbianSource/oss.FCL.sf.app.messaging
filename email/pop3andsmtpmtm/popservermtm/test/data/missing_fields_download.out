STAT
LIST
UIDL
CAPA
RETR 9
QUIT
